src/lv_core/lv_indev.c

```diff
@@ -1127,7 +1134,12 @@ static void indev_drag_throw(lv_indev_proc_t * proc)
         lv_obj_get_coords(drag_obj, &coords_ori);
         lv_coord_t act_x = lv_obj_get_x(drag_obj) + proc->types.pointer.drag_throw_vect.x;
         lv_coord_t act_y = lv_obj_get_y(drag_obj) + proc->types.pointer.drag_throw_vect.y;
-        lv_obj_set_pos(drag_obj, act_x, act_y);
+
+        if(allowed_dirs & LV_DRAG_DIR_HOR)
+			lv_obj_set_x(drag_obj, act_x );
+
+		if(allowed_dirs & LV_DRAG_DIR_VER)
+			lv_obj_set_y(drag_obj, act_y);
```

Comment:
Same here with `lv_obj_set_pos`

Reply:
Did one of your comments get missed? I don't know what is supposed to be changed here.

Reply:
Maybe my first comment "get lost". I meant to write for performance reasons if both directions are enabled `lv_obj_set_pos`  should be used.

Reply:
Sure, I can update this.